src/extension.ts

```diff
@@ -290,6 +312,25 @@ function showStepReferences(clients: Map<string, LanguageClient>): (uri: string,
     };
 }
 
+function showQuickPickItems() {
+    let commandsList = [];
+    commandsList.push({ label: VIEW_REPORT});
+    commandsList.push({ label: RE_RUN_TESTS});
+    commandsList.push({ label: RE_RUN_FAILED_TESTS});
+    return window.showQuickPick(commandsList).then((selected) => {
+        if (selected.label == VIEW_REPORT){
+            var url = "file:///" + getDefaultFolder() + "/reports/html-report/index.html";
+            return opn(url);
+        } else if (selected.label == RE_RUN_TESTS) {
+            return execute(null, { repeat: true, status: "previous run", projectRoot: getDefaultFolder() });
+        } else if(selected.label == RE_RUN_FAILED_TESTS){
```

Comment:
`getDefaultFolder` will give the alphabetically first project from the workspace.  rerun, open reports, and other actions will be performed on that, not on the project which got executed last.